chcemvediet/apps/inforequests/tests/test_cron.py

```diff
@@ -20,6 +20,10 @@
 
 class CronTestCaseMixin(TestCase):
 
+    def _pre_setup(self):
+        super(CronTestCaseMixin, self)._pre_setup()
+        Inforequest.objects.all().delete()
```

Comment:
Naco tu potrebujes premazat vsetky inforequesty? Ved tu nemame ziaden test, co by sa pokazil pri existencii ineho inforequestu, ci?

Reply:
Testy zbehnu, len som nechcel aby sme mali data z "vonku", ktore sme si nevytvorili v rami `test_cron`. Lebo nami vytvoreny default `Inforequest` sa tiez selectne a ide do porovnanie ci sa ma/nema poslat reminder. https://github.com/martinmacko47/chcemvediet/blob/bd39bc4338041c2cae5bfaa2bf3965051f6ade73/chcemvediet/apps/inforequests/cron.py#L71

Reply:
:ok:, odstranil som to

Reply:
IMHO nevadi, ze sa (ne)posle reminder aj pre iny inforequest. Test sa vyslovene pozera, iba ci sa (ne)posle reminder pre dany testovany inforequest.